packages/wp-story-editor/src/components/documentPane/publish/author.js

```diff
@@ -55,10 +55,8 @@ function Author() {
   const [visibleOptions, setVisibleOptions] = useState(null);
 
   useEffect(() => {
-    if (tab === 'document') {
-      loadUsers();
-    }
-  }, [tab, loadUsers]);
+    loadUsers();
+  }, [loadUsers]);
```

Comment:
👍 verified this only runs when the document panel is open _or_ the publish modal is open